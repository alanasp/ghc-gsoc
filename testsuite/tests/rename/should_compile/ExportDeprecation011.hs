{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

module Main where
import ExportDeprecation011_A
main = print func
