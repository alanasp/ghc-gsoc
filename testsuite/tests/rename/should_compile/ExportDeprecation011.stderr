ExportDeprecation011.hs:4:1: warning: [-Wdeprecations (in -Wdefault)]
    Module ‘ExportDeprecation011_A’ is deprecated:
      Self export ExportDeprecation011_A is deprecated

ExportDeprecation011.hs:5:14: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘func’ (imported from ExportDeprecation011_A):
    Deprecated: "Self export ExportDeprecation011_A is deprecated"
